tests/test_chi_ssa_16.py

```diff
@@ -0,0 +1,100 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.chi_ssa_16 import ChiSsa16Spider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "chi_ssa_16.html"),
+    url="https://greektownchicago.org/about/ssa-16/",
+)
+
+spider = ChiSsa16Spider()
+
+freezer = freeze_time("2020-02-06")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
+
+def test_num():
+    assert len(parsed_items) == 51
+
+
+@pytest.mark.parametrize("item", parsed_items)
+def test_title(item):
+    assert item["title"] == "Greektown Special Service Area Tax Commission #16 Public Meeting"
+
+
+@pytest.mark.parametrize("item", parsed_items)
+def test_description(item):
+    descriptionStart = "Special Service Areas (SSA), known as Business Improvement Districts"
+    assert item["description"].startswith(descriptionStart)
+
+
+def test_start():
+    assert parsed_items[0]["start"] == datetime(2020, 1, 23, 14, 0)
+
+
+def test_end():
+    assert parsed_items[0]["end"] is None
+
+
+def test_time_notes():
+    assert parsed_items[0]["time_notes"] == ""
+
+
+def test_id():
+    str0 = "chi_ssa_16/202001231400/x/"\
+        "greektown_special_service_area_tax_commission_16_public_meeting"
+    assert parsed_items[0]["id"] == str0
+    str1 = "chi_ssa_16/202002271400/x/"\
+        "greektown_special_service_area_tax_commission_16_public_meeting"
+    assert parsed_items[1]["id"] == str1
+
+
+def test_status():
+    assert parsed_items[0]["status"] == "passed"
```

Comment:
We should use the status constants `PASSED` and `TENTATIVE` here instead of strings for clarity